### PR TITLE
EVA-545:Feature/regression test

### PR DESCRIPTION
**Pull request by kandaj**

### CODE REVIEW

File: src/js/eva-manager-config.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+ *
+ * EVA is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with EVA. If not, see <http://www.gnu.org/licenses/>.
+ */
+
```
Comment (cyenyxe) [inline src/js/eva-manager-config.js]: From my understanding, this file should replace `eva-manager.js`.

File: tests/mocha/acceptance/config-manager.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+var baseURL = '@@BASE_URL';
```
Comment (cyenyxe) [inline tests/mocha/acceptance/config-manager.js]: From my understanding, this file should replace `config.js`.

**Comment by kandaj** [comment]
suggested changes are made. 
I’m not sure about eva-manager-config.js <https://github.com/EBIvariation/eva-web/pull/29#pullrequestreview-12687462> and  tests/mocha/acceptance/config-manager.js <https://github.com/EBIvariation/eva-web/pull/29#pullrequestreview-12687462> do you want them rename as eva-manager.js and config.js respectively?

Jag.

**Comment by cyenyxe** [comment]
If we already have the configurable classes, would there be any reason to keep the non-configurable ones?

**Comment by kandaj** [comment]
I’ve added  the eva-manager.js <https://github.com/EBIvariation/eva-web/pull/29#pullrequestreview-12687462> and  tests/mocha/acceptance/config.js <https://github.com/EBIvariation/eva-web/pull/29#pullrequestreview-12687462>  in the  gitignore. 
otherwise each time when we deploy for different environments conflicts would come while merging.
I’m not sure if i’ve explained it correctly.

**Pull request closed without merging**